products/consul.md

```diff
@@ -33,7 +33,7 @@ releases:
     latestReleaseDate: 2022-10-19
     releaseDate: 2021-12-14
 -   releaseCycle: "1.10"
-    eol: false
+    eol: true
```

Comment:
```suggestion
    eol: 2022-08-09
```

Prefer using dates, and we can guess the EOL here as v13 release date.

Reply:
Noted.